libs/dal/src/repositories/change/change.schema.ts

```diff
@@ -44,6 +42,16 @@ changeSchema.virtual('user', {
   justOne: true,
 });
 
+changeSchema.index({
+  _environmentId: 1,
+  _parentId: 1,
+  _entityId: 1,
+  createdAt: 1,
+  type: 1,
+  enabled: 1,
```

Comment:
Still not sure about those two, because I am not entirely sure if the compound index will be unique here.

Reply:
Those are not really needed because their cardinality is VERY low (boolean). So suggest removing both type and enabled from there

Reply:
i think i will remove them because the scan should be really fast in those cases.